core/new-gui/src/app/home/component/home.component.html

```diff
@@ -0,0 +1,54 @@
+<div>
+  <div
+    nz-row
+    nzJustify="space-around"
+    nzAlign="middle">
+    <div
+      nz-col
+      nzFlex="auto">
+      <img
+        src="../../../assets/logos/full_logo_small.png"
+        alt="texera-logo"
+        class="logo" />
+    </div>
+    <texera-google-login
+      nz-col
+      nzFlex="300px">
+    </texera-google-login>
+  </div>
+  <div
+    nz-row
+    nzJustify="space-around"
+    nzAlign="middle"
+    style="margin-top: 100px">
+    <div
+      nz-col
+      nzFlex="auto"
+      class="content">
+      <h2>Collaborative Data Analytics Using Workflows</h2>
+      <p>
+        <br />
+        <i>Texera supports scalable computation and enables advanced AI/ML techniques.</i>
+        <br />
+        <i
+          >"Collaboration" is a key focus, and we enable an experience similar to Google Docs, but for data
+          analytics.</i
+        >
+        <br />
+      </p>
+
+      <h4>
+        <a href="https://youtu.be/2gfPUZNsoBs">Demo Video</a>
```

Comment:
I wonder if we are also interested in including the GitHub repo link here. Ideally to show it as a badge. 

Something like [this](https://ghbtns.com):
<img width="694" alt="Screenshot 2022-12-10 at 21 12 52" src="https://user-images.githubusercontent.com/17627829/206887654-888c470b-941e-4881-a105-729f00920953.png">